node/generate-stackbrew-library.sh

```diff
@@ -61,7 +61,7 @@ function generate_library(){
 
 declare -A aliases
 aliases=(
-	[6.5.0]='6 latest'
+	[6.6.0]='6 latest'
 	[6.3.1]='6 latest'
```

Comment:
the second alias is for debian wheezy only since later node versions don't work on wheezy anymore